src/components/MarkDownEditor/extensions/ImageUploadExtension/ImageUploadExtension.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+import MdEditor, { InsertContentGenerator } from 'md-editor-rt';
+import { Paper } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+import { Svg } from 'components';
+import { StyledFlexAllCenter, StyledFlexColumn, StyledTitleSmall } from 'styles/styledComponents';
+import theme from 'styles/theme';
+
+import { StyledMenuItem, StyledMenuList } from '../extensions.styles';
```

Comment:
Why not to use Extensions with upper cased first letter?

Reply:
done